apps/document-generation/.serverless/cloudformation-template-create-stack.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
We can ignore the `.serverless` directory. Let's add it to `.gitignore`

Reply:
Awesome. Will do.